### PR TITLE
InfluxDB Handler

### PR DESCRIPTION
**Pull request by nikkiattea**
Signed-off-by: Nikki Attea <nikki@sensu.io>

## What is this change?

Adds an influx-db event handler.

## Why is this change necessary?

Allows users to configure an event handler which will write sensu metrics to an influx-db of their choice!

## Does your change need a Changelog entry?

Yas.

## Do you need clarification on anything?

Nah.

## Were there any complications while making this change?

Nah.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -239,7 +239,7 @@ docker_commands () {
         build_tool_binary linux amd64 $cmd "tools"
     done
 
-    for cmd in slack; do
+    for cmd in slack influx-db; do
```
Comment (cyphus) [inline build.sh]: This is a small thing, but we should probably make handlers a variable and loop through it where needed in build.sh.

File: handlers/influx-db/main_test.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	event.Metrics = types.FixtureMetrics()
+
+	var apiStub = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		body, _ := ioutil.ReadAll(r.Body)
+		expectedBody := `answer,foo=bar value=42`
+		assert.Contains(string(body), expectedBody)
+		w.WriteHeader(http.StatusOK)
+		_, err := w.Write([]byte(`{"ok": true}`))
+		require.NoError(t, err)
+	}))
```
Comment (cyphus) [inline handlers/influx-db/main_test.go]: This httptest stuff is great!

File: build.sh (hunk 1 above)
Comment (nikkiattea) [reply build.sh]: @cyphus I can probably do that in sh, but not quite sure how to do that for ps.

**Comment by nikkiattea** [comment]
Filed https://github.com/sensu/sensu-go/issues/1379 for Travis failure.

**Pull request merged**